tests/Provider/LaravelServiceProviderTest.php

```diff
@@ -14,7 +14,7 @@ public function testServiceProviderIsWorking()
         );
 
         $provider = $this->app->make('Optimus\ApiConsumer\Provider\LaravelServiceProvider', [
-            $appMock
+            'app' => $appMock
```

Comment:
This change is related to laravel 6, if you don't name the parameters then is not considered to be the app one, so it failed because app could not be found